server/src/game/index.ts

```diff
@@ -85,26 +127,37 @@ export class Game {
       exportedPlayers[playerName] = player.export();
     }
 
+    const exportedLeaders: any[] = [];
+    for (const leaderID in this.leaders) {
+      const leader = this.leaders[leaderID];
+      exportedLeaders.push(leader.export());
```

Comment:
Couldn't this for-loop be replaced with `this.leaders.map((leader) => leader.export())`? In which case it could be in-lined in the `return { ... }` below. (Same for `exportedPlayers`)

Reply:
While it may be possible to do this without a for loop, `this.leaders.map()` won't work as it isn't an array, it's an object.